multinode-demo/fullnode.sh

```diff
@@ -392,6 +387,7 @@ if [[ $node_type != replicator ]]; then
   default_arg --ledger "$ledger_config_dir"
   default_arg --accounts "$accounts_config_dir"
   default_arg --snapshot-path "$snapshot_config_dir"
+  default_arg --snapshot-package-output-path "$SOLANA_RSYNC_CONFIG_DIR"
```

Comment:
this sucks.  I understand, though.  if we’re starting from a snapshot, that should be explicit?